lusidtools/cocoon/instruments.py

```diff
@@ -17,11 +17,19 @@ def prepare_key(identifier_lusid: str, full_key_format: bool) -> str:
     """
     This function prepares the key for the identifier based on whether the full key or just the code is required
 
-    :param str identifier_lusid: The LUSID identifier in either full key format or code only
-    :param bool full_key_format: Whether or not they key needs to be in the full key format
-
-    :return: str: The LUSID identifier in the correct format
+    Parameters
+    ----------
+    identifier_lusid :  str
+                        The LUSID identifier in either full key format or code only
+    full_key_format :   bool
+                        Whether or not they key needs to be in the full key format
+
```

Comment:
What happened to the indentation here?

Reply:
Updated indentation in all cocoon files, not just load from dataframe